Example/CardParts/CardPartAttributedTextViewController.swift

```diff
@@ -0,0 +1,44 @@
+//
+//  CardPartMutableTextViewController.swift
+//  CardParts_Example
+//
+//  Created by jloehr  on 12/11/19.
+//  Copyright © 2019 CocoaPods. All rights reserved.
+//
+
+import Foundation
+import CardParts
+
+class CardPartAttributedTextViewController: CardPartsViewController {
+    
+    let cardPartAttributedTextView = CardPartAttributedTextView(type: .normal)
+    let cardPartImage = CardPartImageView(image: UIImage(named: "cardIcon"))
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+ 
+        let url: String = "https://www.google.com"
```

Comment:
instead of google add a link to CardParts site `https://github.com/intuit/CardParts`